build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -119,7 +119,7 @@ steps:
       set -e
       service xvfb start
     displayName: Start xvfb
-    condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
+    condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'), ne(variables['EXTENSIONS_ONLY'], 'true'))
```

Comment:
It would be better to re-enable the extension unit tests instead...

https://github.com/microsoft/azuredatastudio/blob/fb8021285267c5ec6266f525c6be41006dadb4d4/build/azure-pipelines/linux/sql-product-build-linux.yml#L143

Reply:
@cheenamalhotra can this PR be closed? Extension unit tests were re-enabled in https://github.com/microsoft/azuredatastudio/pull/24985 and the "Azure Data Studio - Extensions" pipeline has been passing since then